src/components/Dialog.jsx

```diff
@@ -18,6 +18,19 @@ class Dialog extends React.Component {
     // }
   }
 
+  stopEvent(e) {
```

Comment:
Couldn't we just use `Utility.stopEvent(e)` ?

Reply:
Sorry, I only _just_ realised the code repetition when I saw `stopEvent()` being repositioned within this compo. Anyway, this should be marked as a minor comment - potential improvements on a later code cleanup.